### PR TITLE
Styling the Fund.Me wizard questionaire walk-through

### PR DESCRIPTION
**Pull request by ainsleywagon**

### CODE REVIEW

**Comment by ainsleywagon** [comment]
screen shot 2014-07-02 at 5 04 21 pm

File: public/src/css/styles.less [hunk 1]
```diff
@@ -128,6 +128,130 @@ p {
...
+  padding: 15px;
+}
+
+// Wizard
+// -------------------------
+.wizard {
+  .steps {
+     ul {
+       li {
+         width: auto;
```
Comment (MrMaksimize) [inline public/src/css/styles.less]: You don't generally need to do nesting unless you have stuff you put in at each level :)

File: public/src/css/styles.less [hunk 2]
```diff
@@ -128,6 +128,130 @@ p {
...
+.btn {
+  background-color: @dark-green;
+  color: @white;
+  font-size: 150%;
+  margin: 20px;
+  padding: 15px;
+}
+
+// Wizard
+// -------------------------
```
Comment (MrMaksimize) [inline public/src/css/styles.less:155]: Awesome way to follow comments style!!! ++

File: public/src/css/styles.less [hunk 3]
```diff
@@ -128,6 +128,130 @@ p {
...
+     ul {
+       li {
+         width: auto;
+       }
+     }
+  }
+}
+
+.wizard {
+  .actions a{
```
Comment (MrMaksimize) [inline public/src/css/styles.less]: I know this is nit picky, but eventually it helps I promise :)  try to keep a space between the a and the {

also you could always do 

```
.actions a {
  padding:10px;
  :hover {
    font-size:19px
  }
}
```

In terms of nesting 

But it's an FYI you don't need to change the nesting unless you wanna :)

File: public/src/css/styles.less [hunk 4]
```diff
@@ -128,6 +128,130 @@ p {
...
+}
+
+.body .control-label {
+    font-size: 24px;
+    color: @navy;
+}
+
+// Landing Page
+// -------------------------
+.landing {
```
Comment (MrMaksimize) [inline public/src/css/styles.less:214]: Also -- try to keep one line of space between the line of ---- and the first selector :)

so 

```
// ------

.landing {
```

File: views/fundmeWizard.jade [hunk 5]
```diff
@@ -3,7 +3,7 @@ include mixins/forms.jade
 extends layout
 
 block content
-  h2.page-header #{title}
+  //h2.page-header #{title}
```
Comment (MrMaksimize) [inline views/fundmeWizard.jade]: I know i'm the one that commented this out, but would you mind removing this comment altogether?  

Gracias :)

**Comment by MrMaksimize** [comment]
@ainsleywagon awesome!!!!!

just made a few minor style comments, if you could update that would be awesome.

Otherwise looks great and ready to go :)

File: public/src/css/styles.less (hunk 3 above)
Comment (ainsleywagon) [inline public/src/css/styles.less]: @MrMaksimize no totally! I actually was doing that but it wasn't working. Idk why, could'e been doing something else wrong.

**Comment by MrMaksimize** [comment]
:+1:  AWESOME!!!

Go @ainsleywagon !!!!

Merging

**Comment by ainsleywagon** [comment]
❤️❤️❤️❤️🇺🇸❤️❤️❤️❤️

**Pull request merged**